app/products/calls/actions/calls.ts

```diff
@@ -363,7 +363,7 @@ export const canEndCall = async (serverUrl: string, channelId: string) => {
         return false;
     }
 
-    return isSystemAdmin(currentUser.roles) || currentUser.id === call.ownerId;
+    return isSystemAdmin(currentUser.roles) || currentUser.id === call.hostId;
```

Comment:
is this a breaking change?

Reply:
Nope, we've sent hostId since 2022-11.

Reply:
Technically, running this against a previous plugin would make it fail because the backend check would be on the owner ID. Not overly worried though.